Composer/packages/client/src/pages/language-generation/code-editor.tsx

```diff
@@ -2,14 +2,18 @@
 // Licensed under the MIT License.
 
 /* eslint-disable react/display-name */
-import React, { useState, useEffect, useMemo, useContext, useCallback } from 'react';
+import path from 'path';
```

Comment:
We should not be using the path module in the client. I know that there is precedent for doing so, but we should not expand the usage.